Automattic/woocommerce-payments
# Update the order of transactions table columns and widths

Fixes #215 

#### Changes proposed in this Pull Request

This PR changes the order of the columns in the transactions table. It moves the info button to the first column, and places the amount, fee, net, and order number directly after the transaction type. 

Additionally, a fixed width has been applied to the info button. It's right padding has been removed to reduce space between it and the date time. The left border of the date time header has been removed. 

Lastly, a minimum width has been applied to the date/time column to prevent the string from wrapping to two lines. 

**Before:**

<img width="1638" alt="Screen Shot 2019-10-09 at 7 47 39 PM" src="https://user-images.githubusercontent.com/4500952/66536022-7a738900-ead0-11e9-9c35-372664e07a87.png">

**After:**

<img width="1697" alt="Screen Shot 2019-10-09 at 8 01 58 PM" src="https://user-images.githubusercontent.com/4500952/66536048-865f4b00-ead0-11e9-95fc-ede2ceca8bca.png">

<img width="612" alt="Screen Shot 2019-10-09 at 8 01 52 PM" src="https://user-images.githubusercontent.com/4500952/66536063-98d98480-ead0-11e9-9e50-36e9f1ee8b69.png">

 

#### Testing instructions



* View the transactions table at various viewport sizes.
* The date/time should not wrap. 

-------------------


Added todo note to make date / time, amount, fee and net sortable. Will open a new issue.
